.github/workflows/build.yaml

```diff
@@ -51,3 +40,62 @@ jobs:
           cache: 'npm'
       - run: npm ci
       - run: npm run build
+      - uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+  e2e:
+    runs-on: ubuntu-latest
+    needs:
+      - build
+      - lint
+      - test
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
```

Comment:
`fetch-depth: 0` is only required when the Git history or other branches is of any relevance. Is this the case here?

Reply:
I could remove it in every job in the build.yaml file.